Oleksii_Fedorchuk/HW18_Oleksii_Fedorchuk/HW18_Oleksii_Fedorchuk/test_human.py

```diff
@@ -0,0 +1,102 @@
+import pytest
+
+
+@pytest.mark.parametrize("name, age, gender", [("Harry", 10, "male")], ids=["Human is created"])
```

Comment:
only one set of data here, why do you need  parametrize?

Reply:
Will be fixed in an update